examples/config-many-files/teams-config.yml

```diff
@@ -0,0 +1,68 @@
+# This configuration file manages the osquery options in Fleet.
+# The `team_name` key determines which team's hosts the spec applies to.
+# The current thinking is that options specified for `team_name` default
```

Comment:
+1 to having default set of options.

Additionally - probably worth thinking about having **multiple** sets of default options (default, as well as overrides for macOS, linux, etc.).

And similarly, the `options` block per-team should probably support overrides per-platform specific to that team. :woman_shrugging:

Reply:
>probably worth thinking about having multiple sets of default options (default, as well as overrides for macOS, linux, etc.)

Agree. The current thinking is that the default options will support platform-specific overrides.
 
> And similarly, the options block per-team should probably support overrides per-platform specific to that team.

Also agree. The team-specific options will also support per-platform overrides.

The current thinking is that overrides specified for each team will only apply to that team. 

For example, if overrides are specified for macOS machines in the default set of options, those overrides will not apply to macOS machines that are assigned to a specific team. 

The same goes for the team-specific options to default options direction. If overrides are specified for Linux machines in team-specific options, these overrides will not apply to hosts that receive the default options (because they're not a member of a specific team)